packages/vue-components/src/annotations/AnnotatePoint.vue

```diff
@@ -0,0 +1,284 @@
+<template>
+  <div>
+    <div v-if="hasBottomText && hasLabel" class="legend-wrapper">
+      <div class="text-header">
+        {{ computedBottomHeader }}
+      </div>
+      <div>
+        {{ content }}
+      </div>
+    </div>
+
+    <div class="annotate-point">
+      <div v-if="isMounted" class="popover-annotation">
+        <div :style="pointPosition">
+          <v-popover
+            v-if="isMounted"
+            :placement="placement"
+            :delay="0"
+            :triggers="triggers"
+            :popper-triggers="triggers"
+            :hide-triggers="triggers"
+            :distance="computeDistance"
+            :disabled="!hasPopover"
+            :skidding="computeSkidding"
+            :arrow-padding="computeArrowPadding"
+            shift-cross-axis
+          >
+            <div class="hover-wrapper" @click.stop>
+              <button class="hover-point" :style="pointStyle">
+                {{ label }}
+              </button>
+              <div class="hover-label" :style="labelStyle">
+                {{ label }}
+              </div>
```

Comment:
Nice spot, will nest the label inside button it makes more sense that way too.

Reply:
Can consider using background opacity instead of overall opacity. In that way only the background of the point will be affected and its children will not inherit any opacity :)